data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,146 @@
+			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: we will only spend a few minutes at top speed before we start decelerating near our destination. I would recommend buckling into the harness."`
+			choice
+				`	"I am looking forward to seeing where we are going."`
+					goto ourdestination
+				`	"How fast does this go?"`
+			`	"The C3-L0 Maglev has a maximum rated velocity of 11D.E8 meters per second, and can sustain that almost indefinitely so long as the track remains powered. In practice, it rarely maintains that velocity for more than a few minutes at a time, as it is a local transport train. We do have some that go faster, but the amount of acceleration required to make use of them on short tracks is not terribly practical." She looks wistful. "Many people are working on solving the inertia problem, but Embers only know when that will result in something practical."`
+			choice
+				`	"Interesting. How long until we arrive?"`
+					goto ourdestination
+				`	"That number seems familiar."`
+				`	"That sounds like the speed of light."`
+			`	Taely smiles. "After all the safety factors, margins for error, and assorted other adjustments were taken into account, someone noticed that the rated value was a fraction of a meter per second above the coefficient of c, so they adjusted it down slightly so that the rated maximum velocity would match the coefficient of the speed of light in a vacuum.`
+			choice
+				`	"Thanks for the explanation."`
+					goto ourdestination
+				`	"What is a coefficient?"`
+			`	She pulls up a hologram with c=11'DE7'84Am/s on it. "Here, I will switch to base 10 to make it easier for you to understand." As she speaks, the number 299'792'458m/s appears on the screen, and then changes to 2.997'924'58x10^8m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.997'924'58 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 'c=1.1DE784Ax16^7m/s'. "So this would be how we typically write the speed of light when we are using it in our research." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
```

Comment:
Is the use of apostrophes to separate digits a hexadecimal thing, Remnant thing, or neither? Some of your notation seems odd here - I can't understand it, and it isn't explained. For context, American English would use commas to separate every three whole digits and periods to separate between whole numbers and decimals. For instance, one thousand five hundred and four hundredths would be "1,500.04" (I promise I'm not trying to be patronizing if it comes off this way, just trying to understand what the intent here is and make sure what I'm thinking is clear).

Reply:
Let's call it a Remnant thing. It's actually a personal habit, because I've had to deal with a lot of places where the , and . and things look too similar and generate confusion. Given that the Remnant are kind of a high-pressure "need to understand stuff at a glance" types, it makes sense that they'd adopt a digit grouping separator that won't generate confusion and erroneous readings.

Reply:
Got it. I don't know if it's just me, but my concern here is that your numbering system is abnormal enough that your explanation that uses it loses its usefulness as an explanation by using something that's not well-known (at least in my experience). For instance, I have no idea what number 2.997'924'58 actually is.

Reply:
I see your point. I think it probably works to the left of the decimal, but it gets really hard to read in the scientific notation. I think that is a case of one or the other, not both.

Reply:
Alright. This has been changed. Added a bunch of spaces to spread it out a bit; and took all the apostrophes out of the scientific notation.